Description: don't ship part of the generated package, since they're unneeded
Author: Julien Puydt
Forwarded: no

--- a/index.js
+++ b/index.js
@@ -154,14 +154,6 @@
 		}
 	}
 	fs.writeFileSync(
-		path.resolve(__dirname, `output/unicode-${version}/README.md`),
-		compileReadMe({
-			'version': version,
-			'dirs': dirMap,
-			'regenerateExample': '<%= set.toString() %>'
-		})
-	);
-	fs.writeFileSync(
 		path.resolve(__dirname, `output/unicode-${version}/index.js`),
 		compileIndex({ 'version': version, 'data': jsesc(dirMap) })
 	);
@@ -169,17 +161,7 @@
 		path.resolve(__dirname, `output/unicode-${version}/package.json`),
 		compilePackage({ 'version': version })
 	);
-	fs.mkdirSync(
-		path.resolve(__dirname, `output/unicode-${version}/.github/workflows`),
-		{
-			recursive: true,
-		}
-	);
 	[
-		'.github/workflows/publish-on-tag.yml',
-		'.gitattributes',
-		'.gitignore',
-		'.npmignore',
 		'decode-property-map.js',
 		'decode-ranges.js',
 	].forEach(function(file) {
